.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
-*.log
+*.log*.json
```

Comment:
gitignore pattterns are not regex as far as i remember. they are more of shell globs. 
Missing enter between the two patterns

Reply:
oh, and by the way, why would you ignore json files? it will mean that even the package.json gets ignored

Reply:
The patterns here do come under shell globs as mentioned in the gitignore documentation. And enter has been given between the two patterns.
For the initial commit they were not ignored but I changed it later after running the project on my system as I did not want the package-lock.json file to be committed(as i googled and many of them suggested that it is not usually done). And thereon any change in the package.json was done directly on GitHub.